src/app/students/page.tsx

```diff
@@ -0,0 +1,16 @@
+// App.js or a parent component
+'use client'
```

Comment:
Seem to need this because if I remove it I get an error about useStudents not being able to be resolved...
Tried to move 'use client' to useStudents where StudentsProvider is but that doesn't seem to work.